core/services/relay/evm/types/types.go

```diff
@@ -2,26 +2,22 @@ package types
 
 import (
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/lib/pq"
 
 	"gopkg.in/guregu/null.v2"
 
-	"github.com/smartcontractkit/chainlink/core/store/models"
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
-type MercuryConfig struct {
-	FeedID          common.Hash   `json:"feedID"`
-	URL             *models.URL   `json:"url"`
-	ServerPubKey    hexutil.Bytes `json:"serverPubKey"`
-	ClientPrivKeyID string        `json:"clientPrivKeyID"`
-}
-
 type RelayConfig struct {
-	MercuryConfig               *MercuryConfig
-	ChainID                     *utils.Big     `json:"chainID"`
-	FromBlock                   uint64         `json:"fromBlock"`
+	ChainID       *utils.Big `json:"chainID"`
+	FromBlock     uint64     `json:"fromBlock"`
+	TransmitterID string     `json:"transmitterID"`
```

Comment:
What is the difference between this and `EffectiveTransmitterAddress`?

Reply:
`EffectiveTransmitterAddress` can be overwritten by an operator forwarder, and is expected to be an ethereum address.

Reply:
Is this different from the `OCR2OracleSpec.TransmitterID`?

Reply:
Not sure if anything upstream blocks this if set with non eth address format but maybe we can get away from fwdr by setting it to  transmitterID in the delegate?

Reply:
Resolved by making `effectiveTransmitterID` that covers both